### PR TITLE
Link to some used snippets

### PR DESCRIPTION
**Pull request by peteruithoven**
Solving https://github.com/atom/snippets/issues/159

### CODE REVIEW

File: book/02-using-atom/sections/02-text.asc [hunk 1]
```diff
@@ -325,6 +325,11 @@ Bam, just fill that bad boy out and you have yourself a snippet. As soon as you
 
 The snippets functionality is implemented in the https://github.com/atom/snippets[atom/snippets] package.
 
+For more examples, see the snippets in the following languare packages:
```
Comment (izuzak) [inline book/02-using-atom/sections/02-text.asc]: What do you think about putting the links in the same line/sentence? I think that'd be a bit easier to read. For example:

```
For more examples, see the snippets in the https://github.com/atom/language-html/blob/master/snippets/language-html.cson[language-html] and https://github.com/atom/language-javascript/blob/master/snippets/language-javascript.cson[language-javascript] packages. 
```

Comment (peteruithoven) [inline book/02-using-atom/sections/02-text.asc]: Even better.

Comment (izuzak) [inline book/02-using-atom/sections/02-text.asc]: :cool: Did you want to make those changes, @peteruithoven ?

**Comment by peteruithoven** [comment]
@izuzak done. 
Github is awesome, just re-edited the file again, totally online.

**Comment by izuzak** [comment]
:zap: Thanks again! 

> Github is awesome, just re-edited the file again, totally online.

The dream is true! :fireworks:

**Pull request merged**